docs/api-docs/in-depth-guides/rail-integration-guide.mdx

```diff
@@ -1,100 +1,167 @@
 ---
 title: 'Integrate Rail Container Tracking Data'
-description: 'This guide provides a comprehensive, step-by-step approach for integrating North American Class-1 rail container tracking data into your systems. Whether you are a shipper or a logistics service provider, this guide will help you track all your rail container via single API.'
+description: 'This guide provides a comprehensive, step-by-step approach for integrating North American Class-1 rail container tracking data into your systems. Whether you are a shipper or a logistics service provider, this guide will help you track all your rail containers via a single API.'
 mode: "wide"
 
 ---
 
 
-## Terminal40 Rail Integrations Overview
+This is a technical article about rail data within Terminal49's API and DataSync.
 
-Terminal49 container tracking platform integrates with all major North American Class-1 railroads, providing comprehensive visibility into rail container movements. Our platform supports the following carriers:
+For a broader overview, including the reasons why you'd want rail visibility and how to use it in the Terminal49 dashboard,
+ [read our announcement post](https://www.terminal49.com/blog/launching-north-american-intermodal-rail-visibility-on-terminal49/).
+
+## Table of Contents
+
+- [Supported Rail Carriers](#supported-rail-carriers)
+- [Supported Rail Events and Data Attributes](#supported-rail-events-and-data-attributes)
+  - [Rail-specific Transport Events](#rail-specific-transport-events)
+  - [Webhook Notifications](#webhook-notifications)
+  - [Rail Container Attributes](#rail-container-attributes)
+- [Integration Methods](#integration-methods)
+  - [Integration via API](#a-integration-via-api)
+  - [Integration via DataSync](#b-integration-via-datasync)
+
+## Supported Rail Carriers
+
+Terminal49 container tracking platform integrates with all North American Class-1 railroads that handle container shipping, providing comprehensive visibility into your rail container movements.
 
-### Supported Rail Carriers
 - BNSF Railway
 - Canadian National Railway (CN)
 - Canadian Pacific Railway (CP)
 - CSX Transportation
-- Kansas City Southern Railway (KCS)
 - Norfolk Southern Railway (NS)
 - Union Pacific Railroad (UP)
 
-By integrating with these carriers, Terminal49 ensures that you have real-time access to critical tracking data, enabling better decision-making and operational efficiency.
+By integrating with these carriers, Terminal49 ensures that you have direct access to critical tracking data, enabling better decision-making and operational efficiency.
+
+## Supported Rail Events and Data Attributes
+
+Terminal49 seamlessly tracks your containers as they go from container ship, to ocean terminal, to rail carrier.
+
+We provide a set of Transport Events that let you track the status of your containers as they move through the rail system.  You can be notified by webhook whenever these events occur.
```

Comment:
I think we should link to 'set of Transport Events'

Reply:
Where would that link?  To #webhook-notifications header, or are you thinking somewhere else?

Reply:
Done ✅